tests/data_context/test_data_context_profilers.py

```diff
@@ -27,7 +27,7 @@ def test_add_profiler(
     profiler_config_with_placeholder_args: RuleBasedProfilerConfig,
 ):
     args = profiler_config_with_placeholder_args.to_json_dict()
-    for attr in ("class_name", "module_name"):
+    for attr in ("class_name", "module_name", "id_"):
         args.pop(attr, None)
```

Comment:
Just popping for the purposes of testing - we'd normally go through schema validation before returning to the user